src/main/resources/project/server/deployApp.pl

```diff
@@ -30,14 +32,37 @@ sub main {
         additional_options
     /);
 
+    my $source_is_url = 0;
+    # e.g. the following format we accept the following:
+    # '--url=https://github.com/electric-cloud/hello-world-war/raw/master/dist/hello-world.war'
+    if ( $params->{warphysicalpath} =~ /^--url=/ ) {
```

Comment:
Why we have a space after ( in this place, while we don't at L50?

Reply:
Formatting changed for this line